js/tests/visual/tab.html

```diff
@@ -150,6 +150,38 @@
     </div>
   </div>
 
+  <h4>Tabs with disabled tab</h4>
```

Comment:
Why did you add a visuel test in addition of a unit test ? Currently visuel tests are usefull to show something impossible to test in unit tests